plugin/pom.xml

```diff
@@ -42,8 +42,8 @@
     </licenses>
     <properties>
         <no-test-jar>false</no-test-jar>
-        <node.version>16.17.0</node.version>
-        <npm.version>8.18.0</npm.version>
+        <node.version>18.13.0</node.version>
+        <npm.version>8.19.3</npm.version>
```

Comment:
Upgrading these to recent versions to take advantage of the existing testing cycle.